mercure.rst

```diff
@@ -529,6 +542,31 @@ These applications will render the content of Mercure updates in real-time.
 Checkout `the dedicated API Platform documentation`_ to learn more about
 its Mercure support.
 
+Testing
+--------
+
+During functional testing there is no need to send updates to Mercure. it's better to disable updates. They will be handled by a stub publisher::
```

Comment:
It’s (uppercase I)

Reply:
Forgot to remove this sentence last time